src/Sulu/Bundle/CoreBundle/SuluCoreBundle.php

```diff
@@ -55,7 +55,5 @@ public function build(ContainerBuilder $container): void
                 'alias'
             )
         );
-
-        $container->addCompilerPass(new JmsObjectConstructorCompilerPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, -1024);
```

Comment:
Was introduce in https://github.com/sulu/sulu/pull/5414 but with https://github.com/sulu/sulu/pull/7235 it should not longer be required